src/apps/packet_filter/conntrack.lua

```diff
@@ -25,30 +25,27 @@
 local ffi = require 'ffi'
 local lib = require 'core.lib'
 
-local ETHERTYPE_IPV6 = 0xDD86
-local ETHERTYPE_IPV4 = 0x0008
+ffi.cdef [[
+   struct _conststruct {
+      static const int ETHERTYPE_IPV4 = 0x0008;
+      static const int ETHERTYPE_IPV6 = 0xDD86;
 
-local IP_UDP = 0x11
-local IP_TCP = 6
-local IP_ICMP = 1
-local IPV6_ICMP = 0x3a
+      static const int IP_UDP = 0x11;
+      static const int IP_TCP = 6;
 
-local ETHERTYPE_OFFSET = 12
+      static const int ETHERTYPE_OFFSET = 12;
 
-local IPV4_SOURCE_OFFSET = 26
-local IPV4_DEST_OFFSET = 30
-local IPV4_PROTOCOL_OFFSET = 23
-local IPV4_SOURCE_PORT_OFFSET = 34
-local IPV4_DEST_PORT_OFFSET = 36
-local IPV4_TCP_FLAGS = 47
+      static const int IPV4_SOURCE_OFFSET = 26;
+      static const int IPV4_PROTOCOL_OFFSET = 23;
+      static const int IPV4_SOURCE_PORT_OFFSET = 34;
 
-local IPV6_SOURCE_OFFSET = 22
-local IPV6_DEST_OFFSET = 38
-local IPV6_NEXT_HEADER_OFFSET = 20 -- protocol
-local IPV6_SOURCE_PORT_OFFSET = 54
-local IPV6_DEST_PORT_OFFSET = 56
-local IPV6_TCP_FLAGS = 67
+      static const int IPV6_SOURCE_OFFSET = 22;
+      static const int IPV6_NEXT_HEADER_OFFSET = 20; // protocol
+      static const int IPV6_SOURCE_PORT_OFFSET = 54;
+   };
+]]
 
+local const = ffi.new('struct _conststruct')
```

Comment:
The name `_conststruct` is being defined in the global FFI namespace here. That seems like a potential clash. Consider either a proper name or an anonymous struct (`ffi.typeof("struct { ... }")`)?